index.js

```diff
@@ -19,5 +26,123 @@ module.exports = {
       return false;
     }
     return true;
+  },
+  _getHostApp: function() {
+    if (!this._findHost) {
+      this._findHost = function findHostShim() {
+        let current = this;
+        let app;
+        do {
+          app = current.app || app;
+        } while (current.parent.parent && (current = current.parent));
+        return app;
+      };
+    }
+
+    return this._findHost();
+  },
+  _treeShakingEmber(tree) {
+    const options = this.app.options[this.name];
+    let treeShakingOptions = {};
+    if (options) {
+      if (options.bundles) {
+        [/*'include', */'exclude'].forEach((clude) => {
```

Comment:
clude's not a word..

Reply:
includeOrExclude?

Reply:
ok..

Reply:
just saying: https://en.wiktionary.org/wiki/clude

Reply:
but yeah ok I don't know what `cludo` is.. lemme update